test/e2e/framework/auth/helpers.go

```diff
@@ -77,8 +77,13 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 	return err
 }
 
-// BindClusterRole binds the cluster role at the cluster scope
-func BindClusterRole(c v1beta1rbac.ClusterRoleBindingsGetter, clusterRole, ns string, subjects ...rbacv1beta1.Subject) {
+// BindClusterRole binds the cluster role at the cluster scope. If RBAC is not enabled, nil
+// is returned with no action.
+func BindClusterRole(c v1beta1rbac.ClusterRoleBindingsGetter, clusterRole, ns string, subjects ...rbacv1beta1.Subject) error {
+	if !isRBACEnabled {
```

Comment:
Any reason for not using `IsRBACEnabled(f)` instead? Seems like there's potential for race conditions otherwise

Reply:
So I didnt have `f` in this context.

Though now that you mention it, I really do need to just add that to the parameter list and update the references. For some reason I was assuming that `IsRBACEnabled` would have been called and so checking the variable would be fine since it was a `Once` type method. However, there is no guarantee it has been run at all, in which case the variable would be false even though RBAC is enabled.

Reply:
That method is actually just looking at the list of cluster roles so I'm going to take a look at it and see about modifying the signature. I don't need to rely on the entire framework to just inspect a list.